zebra-chain/src/parameters/network.rs

```diff
@@ -116,7 +216,8 @@ impl Network {
     pub fn bip70_network_name(&self) -> String {
         match self {
             Network::Mainnet => "main".to_string(),
-            Network::Testnet => "test".to_string(),
+            // TODO: add network name field?
```

Comment:
Isn't this just `cache_name`, without the trailing "net"?

Reply:
Yeah it is, I'll change that to `network_name` and add "net" in the method.

Reply:
Let's check what `zcashd` does for "regtest" first, because I don't think it will be "regtestnet".